test/FunctionalTests/Microsoft.OData.Core.Tests/UriParser/Parsers/SelectExpandParserTests.cs

```diff
@@ -380,19 +380,55 @@ private SelectToken ParseSelectClause(string select)
         public void ParseStarInExpand()
         {
             // Arrange & Act
-            ExpandToken expandToken = this.StarExpandTesting("*", "Persons");
+            ExpandToken expandToken = this.StarExpandTesting("*", "Person");
 
             // Assert
             ExpandTermToken innerExpandTerm = Assert.Single(expandToken.ExpandTerms);
             innerExpandTerm.PathToNavigationProp.ShouldBeNonSystemToken("Friend");
             Assert.Null(innerExpandTerm.PathToNavigationProp.NextToken);
         }
 
+        [Fact]
+        public void ParseStarInExpandWithComplexTypeAsParent()

```

Comment:
>ParseStarInExpandWithComplexTypeAsParent [](start = 20, length = 40)

Do we have any tests for explicit nav props on complex types (as opposed to star)?;  i.e., do we have tests for: $expand=Address/CityNav1 ?